app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk

```diff
@@ -776,7 +776,8 @@
             ]
         }) }}
 
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+         <hr width="1220px;" color="black" size="2">
```

Comment:
Might be worth making this is CSS class if we're using it in multiple places, we tend to avoid styling the HTML elements themselves